scripts/build

```diff
@@ -10,7 +10,9 @@ GO=${GO-go}
 PKG="github.com/rancher/k3s"
 PKG_CONTAINERD="github.com/containerd/containerd"
 PKG_K3S_CONTAINERD="github.com/k3s-io/containerd"
-PKG_CRICTL="github.com/kubernetes-sigs/cri-tools"
+PKG_CRICTL="github.com/kubernetes-sigs/cri-tools/pkg"
+PKG_K8S_BASE="k8s.io/component-base"
+PKG_K8S_CLIENT="k8s.io/client-go/pkg"
```

Comment:
Are we going to backport these changes to other K3s branches or is this going to be a 1.23+ thing?

If not, maybe we should make these specific changes/optimizations in a separate PR that is then backported to the rest of the branches.

Reply:
I think I would like to move this and the other VERSION_ definitions out to versions.yaml across all the branches, just for consistency's sake. I can move them to a separate commit that can be cherry-picked.

Reply:
Moved out to a separate commit for ease of cherry-picking